app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+    validates :name, :phone, presence: true
+    validates :name, :phone, format: { with: /\A[a-zA-Z]+\z/}
```

Comment:
should name and phone have the same format validation?

Reply:
not necessary, I wanted to validate both fields as a string

Reply:
please specify the validations in the issue, so do  you intend to allow names to be set into phone field?